<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SnoopSpy</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>426</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveFileAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>&amp;Action</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>&amp;Option</string>
    </property>
    <addaction name="actionOption"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <widget class="QMenu" name="menuFilter">
     <property name="title">
      <string>Filter</string>
     </property>
     <property name="icon">
      <iconset resource="snoopspy.qrc">
       <normaloff>:/images/htmlhelp.png</normaloff>:/images/htmlhelp.png</iconset>
     </property>
     <addaction name="actionCapture_Filter"/>
     <addaction name="actionDisplay_Filter"/>
     <addaction name="actionWinDivert_Filter"/>
    </widget>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="menuFilter"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionPointer"/>
    <addaction name="actionLinePointer"/>
    <addaction name="actionDelete"/>
    <addaction name="actionBringToFront"/>
    <addaction name="actionSendToBack"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionLogServer"/>
    <addaction name="actionNetClient"/>
    <addaction name="actionNetServer"/>
    <addaction name="actionWebProxy"/>
    <addaction name="actionWebSniff"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAction"/>
   <addaction name="menuOption"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionPointer"/>
   <addaction name="actionLinePointer"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionShowOption"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Node</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QSplitter" name="leftSplitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>256</width>
       <height>384</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="indentation">
       <number>16</number>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Node</string>
       </property>
      </column>
     </widget>
     <widget class="QTreeView" name="treeView">
      <property name="indentation">
       <number>16</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/option.png</normaloff>:/images/option.png</iconset>
   </property>
   <property name="text">
    <string>Option</string>
   </property>
  </action>
  <action name="actionPointer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/pointer.png</normaloff>:/images/pointer.png</iconset>
   </property>
   <property name="text">
    <string>Pointer</string>
   </property>
  </action>
  <action name="actionLinePointer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/linepointer.png</normaloff>:/images/linepointer.png</iconset>
   </property>
   <property name="text">
    <string>LinePointer</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionBringToFront">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/bringtofront.png</normaloff>:/images/bringtofront.png</iconset>
   </property>
   <property name="text">
    <string>BringToFront</string>
   </property>
  </action>
  <action name="actionSendToBack">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/sendtoback.png</normaloff>:/images/sendtoback.png</iconset>
   </property>
   <property name="text">
    <string>SendToBack</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSaveFileAs">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/saveas.png</normaloff>:/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionShowOption">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/showoption.png</normaloff>:/images/showoption.png</iconset>
   </property>
   <property name="text">
    <string>ShowOption</string>
   </property>
  </action>
  <action name="actionCapture_Filter">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/htmlhelp.png</normaloff>:/images/htmlhelp.png</iconset>
   </property>
   <property name="text">
    <string>Capture Filter</string>
   </property>
  </action>
  <action name="actionDisplay_Filter">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/htmlhelp.png</normaloff>:/images/htmlhelp.png</iconset>
   </property>
   <property name="text">
    <string>Display Filter</string>
   </property>
  </action>
  <action name="actionWinDivert_Filter">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/htmlhelp.png</normaloff>:/images/htmlhelp.png</iconset>
   </property>
   <property name="text">
    <string>WinDivert Filter</string>
   </property>
  </action>
  <action name="actionLogServer">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/tools.png</normaloff>:/images/tools.png</iconset>
   </property>
   <property name="text">
    <string>LogServer</string>
   </property>
  </action>
  <action name="actionNetClient">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/tools.png</normaloff>:/images/tools.png</iconset>
   </property>
   <property name="text">
    <string>NetClient</string>
   </property>
  </action>
  <action name="actionNetServer">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/tools.png</normaloff>:/images/tools.png</iconset>
   </property>
   <property name="text">
    <string>NetServer</string>
   </property>
  </action>
  <action name="actionWebProxy">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/tools.png</normaloff>:/images/tools.png</iconset>
   </property>
   <property name="text">
    <string>WebProxy</string>
   </property>
  </action>
  <action name="actionWebSniff">
   <property name="icon">
    <iconset resource="snoopspy.qrc">
     <normaloff>:/images/tools.png</normaloff>:/images/tools.png</iconset>
   </property>
   <property name="text">
    <string>WebSniff</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="snoopspy.qrc"/>
 </resources>
 <connections/>
</ui>
